test/test_helpers.py

```diff
@@ -41,11 +41,14 @@ def cleanup():
 
 
 def fake_script_calls(test_case, clear=False):
-    with open(str(test_case.fake_script_path / 'calls.txt'), 'r+') as f:
-        calls = [line.split(';') for line in f.read().splitlines()]
-        if clear:
-            f.truncate(0)
-        return calls
+    try:
+        with open(str(test_case.fake_script_path / 'calls.txt'), 'r+') as f:
+            calls = [line.split(';') for line in f.read().splitlines()]
```

Comment:
You can do `[line.split(';') for line in f]` and it's more efficient.

Reply:
there was one minor tweak, that 'for line in f' includes the newline character, but 'read().splitlines()' doesn't. But that was easy enough to fix with a strip('\n').